pykrige/compat.py

```diff
@@ -11,12 +10,6 @@
     from sklearn.model_selection import train_test_split
 
     SKLEARN_INSTALLED = True
-    arg_spec = inspect.getfullargspec(GridSearchCV)[0]
-    # https://stackoverflow.com/a/56618067/6696397
-    if "return_train_score" in arg_spec:
-        GridSearchCV = partial(GridSearchCV, return_train_score=True)
```

Comment:
return_train_score was added in 0.19 while latest release is 0.23 so I think it's safe to assume the scikit-learn version used has it.

Generally I'm not keen on this re-exporting of third party modules with different default parameters, I think we should just remove this.

Reply:
That should be ok since we require `sklearn>=0.19`:
https://github.com/GeoStat-Framework/PyKrige/blob/4a974fd5216779e443914201bce424d59a3dc4b8/setup.py#L103